homeassistant/components/eq3btsmart/manifest.json

```diff
@@ -23,5 +23,5 @@
   "iot_class": "local_polling",
   "loggers": ["eq3btsmart"],
   "quality_scale": "silver",
-  "requirements": ["eq3btsmart==1.2.0", "bleak-esphome==1.1.0"]
+  "requirements": ["eq3btsmart==1.2.1", "bleak-esphome==1.1.0"]
```

Comment:
Can we move out this bump into a separate PR?

../Frenck

Reply:
Is this necessary? The only thing that's changed is that I added a missing `py.typed` file to the library, which led `mypy` to fail on the integration. I'd have to include an ugly workaround in the new code and change that back later in order to have `mypy` not fail.

Reply:
Nevermind, I guess we can merge the dependency PR first.

Reply:
#130297

Reply:
> Is this necessary?

Yes, we like to split out things that can be done separately into separate PRs. The PR form you've also explains that.

../Frenck